package.json

```diff
@@ -67,7 +67,9 @@
         "production": "npm start",
         "unlocalize": "grunt unlocalize",
         "test": "grunt test && grunt build-browser-compressed",
-        "start": "http-server -p 8000 --cors"
+        "server": "http-server -p 8000 --cors",
+        "start": "npm run localize && npm run server && npm run unlocalize",
```

Comment:
let's try not to concatenate commands and instead stick to npm script's hooks. So change this to:
```js
"prestart": "npm run localize",
"start": "npm run server",
"poststart": "npm run unlocalize"
```

One thing you might want to make sure is that the `poststart` script is actually run when you terminate the script with `Ctrl-C`.